pkg/config/engine/containerd/option.go

```diff
@@ -30,7 +30,7 @@ type builder struct {
 	configSource         toml.Loader
 	path                 string
 	runtimeType          string
-	useLegacyConfig      bool
+	configVersion        int64
```

Comment:
This is technically the `defaultVersion` since the version in the config file overwrites this.